### PR TITLE
Using document title in tandem with drag and drop

### LINKED ISSUE #15306: Using Document title in tandem with Drag and drop of content blocks results in multiple errors
## 📝 Provide detailed reproduction steps (if any)

1. Open the [demo of the Document title feature](https://ckeditor.com/docs/ckeditor5/latest/features/title.html#demo)
2. Set a title and some other content in the demo
3. Try dragging any of the content and observe the errors in the console

### ✔️ Expected result

You should be able to drag and drop a content block and no errors should appear in the console.

### ❌ Actual result

Multiple errors are thrown and the content cannot be dragged:

https://github.com/ckeditor/ckeditor5/assets/72079603/ea41e576-f511-4392-a103-2230fa5fc117

## 📃 Other details

* Browser: cross-browser
* OS: cross-OS
* First affected CKEditor version: 40.0.0
* Installed CKEditor plugins: `Title`, `Clipboard`

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

### PR DESCRIPTION
**Pull request by illia-stv**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Fix (clipboard): Should not throw errors while using drag and drop with document title. Closes #15306.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: packages/ckeditor5-clipboard/tests/dragdrop.js [hunk 1]
```diff
@@ -2505,4 +2540,39 @@ describe( 'Drag and Drop', () => {
 			.trim()
 			.replace( />\s+</g, '><' );
 	}
+
+	function CustomTitle( editor ) {
```
Comment (Witoso) [inline packages/ckeditor5-clipboard/tests/dragdrop.js:2544]: Out of curiosity, why is this tested against `CustomTitle`, and not [Title](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/features/title.html) plugin?

**Review by pszczesniak — CHANGES_REQUESTED** [review]
After run the manual test `yarn manual -f heading` i get still the same error:

https://github.com/ckeditor/ckeditor5/assets/7074833/a0ef1029-42c5-4b42-b190-5ec5f1bbc156

**Comment by pszczesniak** [comment]
One more thing i noticed, but his can be a follow-up for this (not allow to drop for example an image into `Title`):

https://github.com/ckeditor/ckeditor5/assets/7074833/22ab611d-6bed-4d63-9fd6-c93111b8614f

**Review by pszczesniak — APPROVED** [review]
LGTM - it does what it should :)

**Review by arkflpc — APPROVED** [review]

**Pull request merged**